### ISSUE TITLE
CNN(backbone) + WDRO

### ISSUE THREAD

**Issue by beaumelon** [bisno/WDRO#5]
Can transformer based backbone (i.e., ViT) be applied for training WDRO?

**Comment by bisno** [comment]
Yes, during  training stage, our method substitute the gradient of last hidden layer. You can try ViT by substituting the gradient with the solved gradients.
